plugins/modules/mysql_info.py

```diff
@@ -387,6 +389,18 @@ def get_info(self, filter_, exclude_fields, return_empty_dbs):
             self.__collect(exclude_fields, return_empty_dbs, set(self.info))
             return self.info
 
+    def is_mariadb(self):
+        if self.server_implementation == "mariadb":
+            return True
+        else:
+            return False
+
+    def is_mysql(self):
+        if self.server_implementation == "mysql":
+            return True
```

Comment:
This would be more at home in module_utils/mysql.py

And it's crazy to think we have a variable `db_engine` in the integrations tests but we don't have an equivalent in the modules. If we implement this in Python, we could then use it in the integrations tests.

Reply:
Just moved it to the `module_utils/mysql.py`